t/test-rdata_to_str.c

```diff
@@ -334,18 +334,6 @@ struct test tdata[] = {
 		.expected = "1 foo.example.com. key667=\"hello\"",
 	},
 
-	{ /* appendix D, figure 5 */
```

Comment:
maybe leave in the code under an #ifdef pair with a comment why it does not work now?